en/components/grids_templates/column_resizing.md

```diff
@@ -120,6 +120,53 @@ public onResize(event) {
 ```
 }
 
+#### Resizing columns in pixels/percentages
+
+Depending on the user scenario, the column widths may be defined in pixels, percentages or a mix of both. All these scenarios are supported by the Column Resizing feature. By default if a column does not have width set, it fits the available space with width set in pixels.
+
+This means that the following configuration is possible:
+
+@@if (igxName === 'IgxGrid') {
+```html
+<igx-grid [data]="data" (onColumnResized)="onResize($event)" [autoGenerate]="false">
+    <igx-column [field]="'ID'" width="10%" [resizable]="true"></igx-column>
+    <igx-column [field]="'CompanyName'" width="100px" [resizable]="true"></igx-column>
+    <igx-column [field]="'ContactTitle'" [resizable]="true"></igx-column>
+</igx-grid>
+```
+}
+@@if (igxName === 'IgxTreeGrid') {
+```html
+<igx-tree-grid [data]="data" primaryKey="ID" foreignKey="ParentID" (onColumnResized)="onResize($event)" [autoGenerate]="false">
+    <igx-column [field]="'Title'" [resizable]="true" [width]="'10%'"></igx-column>
+    <igx-column [field]="'HireDate'" [resizable]="true" [width]="'100px'"></igx-column>
+    <igx-column [field]="'Age'" dataType="number" [resizable]="true"></igx-column>
+</igx-tree-grid>
+```
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+```html
+  <igx-hierarchical-grid class="hgrid" [data]="localdata" (onColumnResized)="onResize($event)" [autoGenerate]="false"
+        [height]="'600px'" [width]="'100%'" #hierarchicalGrid>
+        <igx-column field="Artist" [resizable]="true" [width]="'10%'"></igx-column>
+        <igx-column field="GrammyNominations" [resizable]="true" [width]="'100px'"></igx-column>
+        <igx-column field="GrammyAwards" [resizable]="true"></igx-column>
+        ...
+</igx-hierarchical-grid>
+```
+}
+
+>[!NOTE]
+> There is a slight difference in the way resizing works for columns set in pixels and percentages.
+
+**Pixels**
+
+Resizing columns with widths in pixels works by directly adding or subtracting the horizontal amount of the mouse movement from the size of the column.
+
+**Percentages**
+
+Resizing column with widths in percentages, the horizontal amount of the mouse movement in pixels is translated roughly to its percentage amount relative to the grid width. This means that a column may appear to snap to a certain size when resizing ends. Percentages sizes are only integers and there are usually some pixels lost when sizing a column in this way.
```

Comment:
This is no longer accurate - 'Percentages sizes are only integers and there are usually some pixels lost when sizing a column in this way.' As of the latest changes the percentages are floats to increase the resizing accuracy. Some difference between the % and pixel values is still possible.